src/versions.ts

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable */
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+// Note: Leave this file as ES5 js for compatibility with earlier Node.js versions
+'use strict';
```

Comment:
Is this necessary?

Reply:
I will remove. We should be way past "older versions of node"